crm_group_currency/models/crm_lead.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class CrmLead(models.Model):
+    _inherit = "crm.lead"
+
+    group_currency_id = fields.Many2one(
+        "res.currency", compute="_compute_group_currency_id"
```

Comment:
compute is not stored in this case so every time it will be triggered, instead, we can have just default settings for field

Reply:
updated with your suggestion